### ISSUE TITLE
[es5] Results are not shown

### ISSUE THREAD

**Issue by t2ym** [t2ym/compat-table#2]
[es5] Results are not shown

# Root Cause
- Test `Miscellaneous_Enumerable_properties_can_be_shadowed_by_non-enumerables` hangs up with a Permission Error
- t2ym/thin-hook#398 and t2ym/thin-hook#399

# Error Message
```js
[errorReportService] /errorReport.json {
  "context":"/es5/,script@533396",
  "error":"Error",
  "message":"Permission Denied: Cannot access ",
  "name":"typeof:undefined",
  "isStatic":true,
  "isObject":false,
  "property":"length",
  "opType":"W"
}
```

# Fix
- Use `t2ym/thin-hook@0.4.0-alpha.45` 
- Merge the corresponding fixes for t2ym/thin-hook#398 and t2ym/thin-hook#399 to `plugins/policy/*`
- Enable `targetConfig.policy.unchainAcl: true`

# ~~Workaround~~ (obsoleted)
- Comment out `Object.prototype.length = 42` and `delete Object.prototype.length`
```diff
diff --git a/docs/es5/index.html b/docs/es5/index.html
index 7ca2275..356a4fa 100644
--- a/docs/es5/index.html
+++ b/docs/es5/index.html
@@ -7226,7 +7226,7 @@ for (var i in Function) {
 }
 delete Object.prototype.length;
 return result;
-    ">test(function(){try{var asyncTestPassed=global.__asyncPassedFn && __asyncPassedFn("54");try{return Function("asyncTestPassed","\nvar result = true;\nObject.prototype.length = 42;\nfor (var i in Function) {\n    if (i === 'length') {\n        result = false;\n    }\n}\ndelete Object.prototype.length;\nreturn result;\n    ")(asyncTestPassed)}catch(e){asyncTestPassed=global.__strictAsyncPassedFn && __strictAsyncPassedFn("54");return Function("asyncTestPassed","'use strict';"+"\nvar result = true;\nObject.prototype.length = 42;\nfor (var i in Function) {\n    if (i === 'length') {\n        result = false;\n    }\n}\ndelete Object.prototype.length;\nreturn result;\n    ")(asyncTestPassed)&&"Strict"}}catch(e){return false;}}());
+    ">test(function(){try{var asyncTestPassed=global.__asyncPassedFn && __asyncPassedFn("54");try{return Function("asyncTestPassed","\nvar result = false;\n/*Object.prototype.length = 42;*/\nfor (var i in Function) {\n    if (i === 'length') {\n        result = false;\n    }\n}\n/*delete Object.prototype.length;*/\nreturn result;\n    ")(asyncTestPassed)}catch(e){asyncTestPassed=global.__strictAsyncPassedFn && __strictAsyncPassedFn("54");return Function("asyncTestPassed","'use strict';"+"\nvar result = true;\nObject.prototype.length = 42;\nfor (var i in Function) {\n    if (i === 'length') {\n        result = false;\n    }\n}\ndelete Object.prototype.length;\nreturn result;\n    ")(asyncTestPassed)&&"Strict"}}catch(e){return false;}}());
 </script></td>
 <td class="unknown" data-browser="es5shim">?</td>
 <td class="unknown obsolete" data-browser="konq4_13">?</td>
```

**Comment by t2ym** [comment]
Patched for GitHub Pages

**Comment by t2ym** [comment]
Fixed with `thin-hook@0.4.0-alpha.45`
